### PR TITLE
Implement two essential operations sendInitialQuery and getPage

### PR DESCRIPTION
**Pull request by DemyanchikVadim**
Implement two essential operations setInitialQuery and getPage #2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+const j = request.jar();
+
+j.setCookie(
+  request.cookie(`sessionalis=${SESSION_ID}`),
+  'http://86.57.174.45/pls/alis/StartEK/index.php'
+);
+
+function sendInitialQuery(query, callback) {
+  const START_URL = `http://86.57.174.45/alis/EK/do_searh.php?radiodate=simple&valueINP=${query.year}&tema=ALL&tag=ALL`;
```
Comment (irnc) [inline index.js]: `86.57.174.45` shouldn't be hardcoded, as different libraries use different hosts.

Comment (irnc) [inline index.js]: Please adjust `tag` when querying by year.

I guess there should be a map from query properties to tags. While initially you implement only query by year, other queries should explicitly fail.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    if (err) {
+      callback(err);
+      return;
+    }
+    callback(null, body);
+  });
+}
+
+function getPage(url, callback) {
+  request({ url, jar: j }, (err, response, body) => {
```
Comment (irnc) [inline index.js]: `j` is used from lexical scope here. Let's pass it as an argument to simplify testing.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    if (err) {
+      callback(err);
+      return;
+    }
+    callback(null, body);
+  });
+}
+
+sendInitialQuery({ year: 2016 }, (err, body) => {
```
Comment (irnc) [inline index.js]: That code should be placed outside library module. When you would export your functions, you could use them from outside.

File: index.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+const request = require('request');
+const SESSION_ID = require('./config.json').SESSION_ID;
```
Comment (irnc) [inline index.js]: This module is planned as a library, so configuration should be provided on call time, not from static file.

File: .eslintrc [hunk 5]
```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "airbnb",
+  "rules": {
+    "no-console": 0
```
Comment (irnc) [inline .eslintrc]: This is a good rule to prevent debugging code to be merged upstream. Could we have it back? ;)

Comment (DemyanchikVadim) [inline .eslintrc]: Check!

File: index.js (hunk 3 above)
Comment (DemyanchikVadim) [inline index.js]: Check!

File: index.js (hunk 2 above)
Comment (DemyanchikVadim) [inline index.js]: Check!

File: .gitignore [hunk 6]
```diff
@@ -1,2 +1,3 @@
 .idea
 node_modules
+/config.json
```
Comment (irnc) [inline .gitignore]: There is no `config.json`, so no need to ignore it.

**Pull request closed without merging**